### ISSUE TITLE
Create mediaclip returns incorrect XML

### ISSUE THREAD

**Issue by RobertBoes** [bluebillywig/composer-vmsrpc#1]
Hi,

Incorrect XML is returned when creating a publication. According to the tests (RPCTest::testXmlCreateDeleteMediaClip) it should return
```xml
<response code="" error="" id="">
  <messages>
    <message time="" context=""></message>
  </messages>
</response>
```

But the returned response from bbvms.com is:
```xml
<?xml version="1.0"?>
<response code="200" error="false" id="3484861">mediaclip was saved
	<messages>
		<message time="Mon, 11 Nov 19 13:57:33 +0100" context="unavailable">no media info available for this media-clip</message>
		<message time="Mon, 11 Nov 19 13:57:33 +0100" context="unavailable">no media info available for this media-clip</message>
	</messages>
</response>
```

So it returns two messages with the same content. This causes the unittests to fail.

**Comment by wmrsp** [comment]
Hi @RobertBoes,

Thank you for reporting this. The format of the xml response was changed in a recent update to the VMS API. This cause the test to expect a different xml format.

We fixed this is the most recent release 0.97.2 by disregarding the `<messages>` tag all together and only check the `<response>` tag. This is all that is needed for the test anyways since that is the tags that holds the respons status code.

Greetings,
Wimar Schippers
Blue Billywig Dev Team

**Comment by RobertBoes** [comment]
Hey @wmrsp,

Tests are failing, since the PHP requirement was updated to 7.1, the lower versions should be removed from `.travis.yml`. Changing the required PHP version to `>=7.1.27` seems good, since the version that's used on Travis is 7.1.27. I don't really know why it's not using the latest 7.1 version though..

I also think wrong username/password credentials are used, because `BlueBillywig\VMSRPC\RPCTest::testCanBeCreated` is failing with the message: login failed, wrong credentials?

I can make a PR for the PHP version requirements if you'd like.

**Comment by wmrsp** [comment]
Hi @RobertBoes,

Thank you for the helping hand :). First time Travis user here. I think it's a great addition to our workflow.

I lowered the PHP requirement to 7.1.27 in order to facilitate Travis so we got that sorted now.
I notice that indeed the login fails for some reason still. We will have a look why that is. I suspect that maybe the login credentials I am using only have access from our office and I might need to use other ones.
